### DISCUSSION TITLE
Воспроизведение звуковых файлов по нажатии F3

### DISCUSSION THREAD

**Discussion by native-engine** [doublecmd/doublecmd#1244] [Suggestions]
Здравствуйте! А Вы не думали добавить в дистрибутив какой-нибудь простенький проигрыватель звуковых файлов и вызывать его для воспроизведения звуковых файлов по нажатии на клавишу F3? А то очень неудобно выбирать звуки из большой библиотеки в программах для работы со звуком, Sound Forge хотя бы позволял это делать прямо в диалоговом окне открытия файла, не говоря уже о Total Commander, а под Линуксом я до сих пор ничего вменяемого для этой цели не нашёл :)

**Comment by alexx2000** [comment]
Здравствуйте, для этого есть несколько плагинов [wlxmpv](https://gitlab.com/bayarookie/wlxmpv/-/tree/master), [gstplayer](https://github.com/j2969719/doublecmd-plugins/tree/master/plugins/wlx/gstplayer).

**Comment by native-engine** [comment]
Не знал :) А как их установить? Добавил в настройках первый, не работает, хотя состояние отображает как "+". В настройках плагина MP3 присутствует, но почему-то по F3 продолжает отображаться в текстовом редакторе )))

**Reply by Skif-off** [reply]
Плагин использует mpv, эта программа установлена?

P.S. gstplayer может быть проще: необходимые ему библиотеки скорее всего уже есть в системе.

**Reply by native-engine** [reply]
А который из них легче и быстрее запускается?

**Reply by Skif-off** [reply]
Если правильно понимаю, плагин после первого запуска остаётся висеть в памяти, вместе с потребными библиотеками, тогда теоретически gstplayer может быть побыстрее, т.к. wlxmpv на каждый файл запускает mpv. С другой стороны, mpv, в принципе, достаточно быстрая штука и это может быть некритично.
Практичнее, наверное, самостоятельно сравнить и определиться окончательно :)

К слову, gstplayer только для GTK2-версии, для Qt5-версии можно ещё [qmediaplayer_qt](https://github.com/j2969719/doublecmd-plugins/tree/master/plugins/wlx/qmediaplayer_qt) попробовать.

P.S. Сам wlxmpv пока так и не пощупал, привык к [этому](https://github.com/j2969719/doublecmd-plugins/tree/master/plugins/wlx/mpv).
